cluster/saltbase/salt/kube-master-addons/init.sls

```diff
@@ -35,3 +35,4 @@ kube-master-addons:
     - restart: True
     - watch:
       - file: master-docker-image-tags
+      - file: /etc/kubernetes/kube-master-addons.sh
```

Comment:
@dchen1107 so that the service can restart if we have to load more docker images.